server/views/pages/contacts/manage/contactConfirmation/confirmation.njk

```diff
@@ -77,7 +77,9 @@
                 <div class="govuk-tabs__panel govuk-!-static-padding-top-7 govuk-tabs__panel--hidden" data-qa="restrictions-result-message" id="restrictions">
 
                     {% if globalRestrictions.length >0 %}
-                        {{ restrictionCard(globalRestrictions, contact) }}
+                        {% set heading = 'Global restrictions for contact ' + ((contact.firstName + " " + contact.lastName) | capitaliseName ) %}
```

Comment:
not sure what the issue here, i mean this capitaliseName is simple function and works as per the requirement here, 
`((contact.firstName + " " + contact.lastName) | capitaliseName `

formatNameFirstNameFirst is bit more advance functionality and finally called into capitaliseName method and pass the generated name into it. so both ideally should work but this seems more straight forward to me

Reply:
Because using 1 filter for all names in titles means we can update the format easily in 1 place. It's used in every page title but here (and I see in a couple of places Rosario did that I missed)